clients/src/main/java/org/apache/kafka/clients/consumer/ConsumerRebalanceListener.java

```diff
@@ -119,15 +119,15 @@
 public interface ConsumerRebalanceListener {
 
     /**
-     * A callback method the user can implement to provide handling of offset commits to a customized store.
+     * A callback method which users can implement to provide handling of offset commits for a customized store.
```

Comment:
uhm here I find the change is not really making things better. I wouldn't necessarily change this, but if we think it should be improved maybe just `A callback method the user can implement to provide handling of offset commits **sent to** a customized store`